src/test/crimson/seastore/test_seastore.cc

```diff
@@ -509,6 +550,54 @@ TEST_F(seastore_test_t, attr)
     bl2 = test_obj.get_attr(*seastore, SS_ATTR);
     decode(ss_str2, bl2);
     EXPECT_EQ(ss_str, ss_str2);
+
+    bl2.clear();
+    oi_str2.clear();
+    bl2 = test_obj.get_attr(*seastore, OI_ATTR);
+    decode(oi_str2, bl2);
+    EXPECT_EQ(oi_str, oi_str2);
+
+    test_obj.rm_attr(*seastore, OI_ATTR);
+    test_obj.rm_attr(*seastore, SS_ATTR);
+
+    attrs = test_obj.get_attrs(*seastore);
+    bl2.clear();
+    bl2 = attrs[OI_ATTR];
+    EXPECT_EQ(bl2.length(), 0);
+    bl2.clear();
+    bl2 = attrs[SS_ATTR];
+    EXPECT_EQ(bl2.length(), 0);
+  }
+  {
+    std::string oi("asdfasdfasdf");
+    bufferlist bl;
+    encode(oi, bl);
+    test_obj.set_attr(*seastore, OI_ATTR, bl);
+
+    std::string ss("f");
+    bl.clear();
+    encode(ss, bl);
+    test_obj.set_attr(*seastore, SS_ATTR, bl);
+
+    std::string test_val("ssssssssssss");
+    bl.clear();
+    encode(test_val, bl);
+    test_obj.set_attr(*seastore, "test_key", bl);
+
+    test_obj.rm_attr(*seastore, OI_ATTR);
+    test_obj.rm_attr(*seastore, SS_ATTR);
+    test_obj.rm_attr(*seastore, "test_key");
+
+    auto attrs = test_obj.get_attrs(*seastore);
+    bufferlist bl3 = attrs[OI_ATTR];
+    EXPECT_EQ(bl3.length(), 0);
```

Comment:
This is ambiguous, could either be an OI_ATTR of length 0 or a missing oi attr.  Instead, do
```
EXPECT_NE(attrs.find(OI_ATTR), attrs.end());
```

Reply:
Change the other instances in this PR as well.